docs/api/clipboard.md

```diff
@@ -28,56 +22,122 @@ The `clipboard` module has the following methods:
 
 ### `clipboard.readText([type])`
 
-* `type` String (optional) - Can be `selection` or `clipboard`. `selection` is only available on Linux.
+* `type` String (optional) - Can be `selection` or `clipboard`. Default is 'clipboard'.
```

Comment:
I think it is important to keep the qualifier here that selection is only available on Linux instead of putting that in the description.
```suggestion
* `type` String (optional) - Can be `selection` or `clipboard`. `selection` is only available on Linux. Default is 'clipboard'.
```

Reply:
i'll undo that change!